src/main/java/com/incognia/onboarding/RegisterSignupRequest.java

```diff
@@ -17,4 +19,5 @@ public class RegisterSignupRequest {
   String policyId;
   String accountId;
   List<AdditionalLocation> additionalLocations;
+  Map<String, Object> customProperties;
```

Comment:
I guess we should add some asserts like https://github.com/inloco/incognia-api-java/blob/dc8fca3ee23331ec90ee32a85d062603b8bc43a0/src/test/java/com/incognia/api/clients/TokenAwareDispatcher.java#L141 to validate that custom properties is being sent correctly

Reply:
Done!